ios/build.sh

```diff
@@ -136,6 +136,10 @@ update_bundle ( ) {
 # First things first - update our dependencies.
 update_bundle
 
+echo
+echo "Load dependencies with Carthage"
+carthage update --platform iOS
```

Comment:
This should probably be `carthage bootstrap` instead of `carthage update`. `bootstrap` will pull in the resolved dependencies but `update` will get new versions and write it in `Cartfile.resolved`. The build agents should use the resolved dependencies so that we get reproducible builds.

Also would be nice to fail the build if carthage is missing or it fails to build dependencies. So maybe add something like `|| fail "carthage boostrap failed"`.